index.js

```diff
@@ -9,6 +9,7 @@ module.exports = function(config) {
     var validateConfigs = require('./lib/validate_configs');
     var name = config.name ? config.name : 'terafoundation';
     var loggerClient = require('./lib/logger_utils').loggerClient;
+    var state_connection = 'default';
```

Comment:
This is configuring logging. State is a concept from teraslice that doesn't exist in the context of terafoundation.